src/com/richard/walker/weatherstation/Main.java

```diff
@@ -0,0 +1,29 @@
+package com.richard.walker.weatherstation;
+
+import com.richard.walker.weatherstation.domain.WeatherStation;
+import com.richard.walker.weatherstation.vendor.*;
+
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+        StandardRegionalWeather regionalWeather = new StandardRegionalWeatherImpl();
+        NoaaRegionalWeather noaaRegionalWeather = new NoaaRegionalWeatherImpl();
+
+        StandardRegionalWeatherAdapter regionalWeatherAdapter = new StandardRegionalWeatherAdapter(noaaRegionalWeather);
```

Comment:
We should return the interface, not the Adapter. It helps illustrate the conversion